cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -14,18 +14,25 @@ if [ -n "${KUBEVIRTCI_TAG}" ] && [ -n "${KUBEVIRTCI_GOCLI_CONTAINER}" ]; then
     echo "WARNING: KUBEVIRTCI_GOCLI_CONTAINER is set and will take precedence over the also set KUBEVIRTCI_TAG"
 fi
 
-if [ "${KUBEVIRTCI_RUNTIME}" = "podman" ]; then
-    _cli="pack8s"
+if docker ps >/dev/null; then
+    _cri_bin=docker
+    echo "selecting docker as container runtime"
+elif podman ps >/dev/null; then
+    _cri_bin=podman
+    echo "selecting podman as container runtime"
```

Comment:
Can we keep an option for users to explicitly set cri checking if KUBEVIRTCI_RUNTIME has being set ?

Reply:
Yes.

Reply:
Done.